doc/pages/JenkinFeatures.md

```diff
@@ -0,0 +1,145 @@
+
+## Grouping and Granularity
+We have many tests, so it is important to be able to slice and dice them in different ways to be efficient with testing, debugging, and triage.
+Tests are organized into groups typically based on where they came from and what type of test they are: `openjdk`, `perf`, `system`,  `functional`, `external`.
+
+## Input Parameters
+AQAvit test job parameters are grouped logically by the type of input they are.
+
+### Test Repositories Parameters
+Repositories where we pull test material from. Unless you are testing test code, these do not need to be changed.
+
+| Parameter                             | Description                                                                                               |
+|--------------------------------------|-----------------------------------------------------------------------------------------------------------|
+| `ADOPTOPENJDK_REPO`                   | Optional. Your fork of `aqa-tests`.                                                                      |
+| `ADOPTOPENJDK_BRANCH`                 | Optional. Your branch off of your fork of `aqa-tests`.                                                  |
+| `OPENJ9_REPO`                         | Optional. Your fork of `openj9`.                                                                         |
+| `OPENJ9_BRANCH`                       | Optional. Your branch of your fork of `openj9`.                                                         |
+| `OPENJ9_SHA`                          | Optional. Pin to a specific SHA of `openj9`.                                                            |
+| `JDK_REPO`                            | Optional. Use test material from a specific OpenJDK repository.                                         |
+| `JDK_BRANCH`                          | Optional. Use test material from a specific OpenJDK branch.                                             |
+| `OPENJDK_SHA`                         | Optional. Pin to a specific OpenJDK SHA.                                                                |
+| `TKG_OWNER_BRANCH`                    | Optional. Use a specific `adoptium/TKG` fork/branch.                                                    |
+| `ADOPTOPENJDK_SYSTEMTEST_OWNER_BRANCH`| Optional. Use a specific `adoptium/aqa-systemtest` fork/branch.                                         |
+| `OPENJ9_SYSTEMTEST_OWNER_BRANCH`       | Optional. Use a specific `openj9/openj9-systemtest` fork/branch.                                        |
+| `STF_OWNER_BRANCH`                    | Optional. Use a specific `adoptium/STF` fork/branch.                                                    |
+| `JCK_GIT_REPO`                        | Optional. Use a specific private repository for JCK test material supplied under OCTLA.                |
```

Comment:
We should add a column to the table to include the suggested default settings for parameters.  Some of the descriptions need updating.  Some parameters are not optional, but rather there are default values, so one does not need to fill in the value, because a default value is set.